### DISCUSSION TITLE
add an option for z axis

### DISCUSSION THREAD

**Discussion by pvtex** [OpenStickCommunity/GP2040-CE#1205] [Ideas]
Hello,

it would be great if there would be an option for a z axis.

As example:
      i would like to connect a 3 axis analog joystick with a x and y axis and a z axis (twist the joystick)

That would be great for many flight games and other scenorios.
you could use that 3 axis joystick as a PTZ cam Controller for controlling pan, tilt and zoom.

regards 
Bjoern

**Comment by TheTrainGoes** [comment]
Hi everyone,

This is general notice that we are closing down the discussion section of the GP2040-CE repo.  

If you would like to discuss anything further please open an issue or join our Discord:
https://discord.com/servers/openstickcommunity-1049366310389289001
